connect-inject/envoy_sidecar_test.go

```diff
@@ -203,3 +204,187 @@ func TestHandlerEnvoySidecar_NamespacesAndAuthMethod(t *testing.T) {
 && /consul/connect-inject/consul logout \
   -token-file="/consul/connect-inject/acl-token"`)
 }
+
+func TestHandlerEnvoySidecar_Resources(t *testing.T) {
+	mem1, err := resource.ParseQuantity("100Mi")
+	require.NoError(t, err)
+	mem2, err := resource.ParseQuantity("200Mi")
+	require.NoError(t, err)
+	cpu1, err := resource.ParseQuantity("100m")
+	require.NoError(t, err)
+	cpu2, err := resource.ParseQuantity("200m")
+	require.NoError(t, err)
+	zero, err := resource.ParseQuantity("0")
+	require.NoError(t, err)
+
+	cases := map[string]struct {
+		handler      Handler
+		annotations  map[string]string
+		expResources corev1.ResourceRequirements
+		expErr       string
+	}{
+		"no defaults, no annotations": {
+			handler:     Handler{},
+			annotations: nil,
```

Comment:
I think you don't need to set these if they are zero values, unless you wanted to be explicit.

Reply:
Yeah was trying to be explicit